content/ja/docs/concepts/architecture/nodes.md

```diff
@@ -294,6 +294,26 @@ Message:        Node is shutting, evicting pods
 
 {{< /note >}}
 
+##  ノードの非正常終了 {#non-graceful-node-shutdown}
+
+{{< feature-state state="alpha" for_k8s_version="v1.24" >}}
+
+コマンドがkubeletのinhibitor locksメカニズムをトリガーしない場合や、ShutdownGracePeriodやShutdownGracePeriodCriticalPodsが適切に設定されていないといったユーザーによるミス等が原因で、ノードがシャットダウンしたことをkubeletのNode Shutdownマネージャーが検知できないことがあります。詳細は上記セクション [Graceful Node Shutdown](#graceful-node-shutdown) をご参照ください。
+
+ノードのシャットダウンがkubeletのNode Shutdownマネージャーに検知されない場合、StatefulSetを構成するpodはシャットダウン状態のノード上でterminating状態のままになってしまい、他の実行中のノードに移動することができなくなってしまいます。これは、ノードがシャットダウンしているため、その上のkubeletがpodを削除できず、それにより、StatefulSet が新しいpodを同じ名前で作成できなくなってしまうためです。podがボリュームを使用している場合、VolumeAttachments はシャットダウン状態のノードによって削除されないため、podが使用しているボリュームは他の実行中のノードにアタッチすることができなくなってしまいます。その結果として、StatefulSet状で実行中のアプリケーションは適切に機能しなくなってしまいます。シャットダウンしていたノードが復旧した場合、そのノード上のpodはkubeletに削除され、他の実行中のノード上に作成されます。また、シャットダウン状態のノードが復旧できなかった場合は、そのノード上のpodは永久にterminating状態のままとなります。
+
+上記の状況を脱却するには、ユーザが手動で `NoExecute` または `NoSchedule` effect を設定して `node.kubernetes.io/out-of-service` taintをノードに付与することで、故障中の状態に設定することができます。`kube-controller-manager` において `NodeOutOfServiceVolumeDetachfeature`[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) が有効になっており、かつノードがtaintによって故障中としてマークされている場合は、ノードに一致するtolerationがなく、ノード上のterminating状態のpodに対するボリュームデタッチ操作がなければ、当該ノード上のpodが強制的に削除されます。
+
+non-graceful shutdownの間に、podは以下の2段階で終了します:
+
+1. 一致する `out-of-service` tolerationを持たないpodを強制的に削除する。
+2. 上記のpodに対して即座にボリュームでタッチ操作を行う。
+
+{{< note >}}
+- `node.kubernetes.io/out-of-service` taintを付与する前に、ノードがシャットダウンしているか電源がオフになっていることを確認してください（再起動中ではないこと）。
+- podの別ノードへの移動後、シャットダウンしていたノードが回復した場合はユーザが手動で out-of-service taintを削除する必要があります。
```

Comment:
Thank you for your comments.
Exactly, I didn't translate the sentence "since the user was the one who originally added the taint." because it's obvious.
If we want to translate truely to the original as much as possible, how about below translation?
```
Podの別ノードへの移動後、シャットダウンしていたノードが回復した場合は、ユーザが手動で付与した out-of-service taintをユーザ自ら手動で削除する必要があります。
```

Reply:
+1
Your suggestion looks good to me.
Could you update PR?

Reply:
done